generate/normalize_benchmark_config.sh

```diff
@@ -40,4 +40,4 @@ while read config; do
 			continue
 		fi
 	done
-done <../generate/benchmark_config.txt
+done <examples/benchmarks/config_"$BACKEND".txt
```

Comment:
Nitpick: As a shell noob, I'd think that this would be somehow better as the version in `generate/benchmarks.sh`, i.e.
```suggestion
done <"examples/benchmarks/config_$BACKEND.txt"
```
Is it equivalent?

Reply:
Yes, they are equivalent. In the second case, I would typically write `${BACKEND}` instead, since now the text after the variable could be part of it (not here, because of the `.` in `.txt`). I prefer this version, but it doesn't really matter here.

(it should be consistent though, you're right)